packages/backend/.eslintignore

```diff
@@ -0,0 +1 @@
+dist/
```

Comment:
Lint error

Reply:
That is kind of bug in `eslint-typescript`. It helps to prevent eslint from linting the `dist/` folder which it strangely does sometimes.